src/commands/Initialize/test/validatorAppType.test.ts

```diff
@@ -0,0 +1,177 @@
+import { describe, expect, test } from '@jest/globals';
```

Comment:
What's the difference of validator.test.ts?

Reply:
I changed validatorAppType to appType:
https://github.com/kintone-labs/kintone-cli/pull/147/commits/5fef77df965e2e1e24384101ace551be0d5158e7

Reply:
Is it ok to remove an invalid app type case??
https://github.com/kintone-labs/kintone-cli/commit/dbcd75298c47fd3a139ff08693a309cb516d1287

Reply:
> Is it ok to remove an invalid app type case?? [dbcd752](https://github.com/kintone-labs/kintone-cli/commit/dbcd75298c47fd3a139ff08693a309cb516d1287)

It's OK, the invalid app type case is updated here:
https://github.com/kintone-labs/kintone-cli/pull/112/commits/8db89762416dd03b6ec6bf44091d3e2e7fb0988c#diff-e0073e61c60ec0ba6020821f942dcabe5b6f151e029ec3509649df190576f541

Reply:
Got it.